controllers/sessions.js

```diff
@@ -13,10 +13,10 @@ function checkLoggedIn(req, res, next) {
 
 
 
-router.route('/')
-    .get( (req, res) => {
-        
-        res.json({email: req.session.email, name: req.session.name})
+        res.json({
+            user_id: req.session.user_id,
+            email: req.session.email
```

Comment:
will need to add 

name: req.session.name 

for the rendering of the name instead of the signin button in the header